pkg/components/exporter/exporter.go

```diff
@@ -0,0 +1,341 @@
+package exporter
+
+import (
+	"strconv"
+
+	"github.com/sustainable.computing.io/kepler-operator/pkg/api/v1alpha1"
+	"github.com/sustainable.computing.io/kepler-operator/pkg/components"
+	"github.com/sustainable.computing.io/kepler-operator/pkg/utils/k8s"
+
+	securityv1 "github.com/openshift/api/security/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+)
+
+const (
+	prefix = "kepler-exporter-"
+
+	SCCName = prefix + "scc"
+
+	ServiceAccountName   = prefix + "sa"
+	FQServiceAccountName = "system:serviceaccount:" + components.Namespace + ":" + ServiceAccountName
+
+	ClusterRoleName        = prefix + "cr"
```

Comment:
Can we make it ClusterRoleName = prefix + "clusterrole"?

Reply:
Actually I was thinking the other way around to remove the suffix completely because `kind` is already a good enough differentiator.

Reply:
removing suffix is also fine but if we go with it then I suggest we should mention it completely.